ml-proto/TestingTodo.md

```diff
@@ -22,7 +22,7 @@ Operator semantics:
  - ~~test that shifts don't mask their shift count. 32 is particularly nice to test.~~
  - test that `page_size` returns a power of 2
  - ~~test that arithmetic operands are evaluated left-to-right~~
- - test that call and store operands are evaluated left-to-right too
+ - ~~test that call and store operands are evaluated left-to-right too~~
```

Comment:
One additional test we can add here is to test that `call_indirect` evaluates its call operand before the argument operands. This is probably best done after #136 is in though, so feel free to just add it to TestingTodo.md for now.